recipes/ofeli/all/test_package/conanfile.py

```diff
@@ -0,0 +1,20 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def config_options(self):
+        self.settings.compiler.libcxx = "libstdc++11"
```

Comment:
```suggestion
```
Same here

Reply:
I just made the changes, I am waiting for the CI to finish. Thanks for your response!

Reply:
Bump

Reply:
Without this config option the test don't works, is not correctly have this here? What i could do for solve this? greetings

Reply:
Thats a good sign the binaries are not being compiled with the expected conan settings.

It seems the project hardcodes it to this https://github.com/rtouzani/ofeli/search?l=Makefile&q=std (i didnt check all 17 pages but I assume it's everywhere)

Since you already limited this to gcc [here](https://github.com/conan-io/conan-center-index/pull/8375/files#diff-1707c887febaab51f38cd2d4e92ee705c47eeabbe5fc8a6e5fe80c8578beb9afR34-R36)
 you can also limit the `libcxx` setting as well

Reply:
Please remove this , and apply my other suggestions. Ofeli actually ignores `CPPFLAGS` provided by conan : https://github.com/rtouzani/ofeli/blob/524b2cc3ed51fa984c9a10d685828f41c3d7a2e4/configure.ac#L47 , which makes it use the new libstdc++ abi by default